Alpaca.Markets/RestClient.Polygon.cs

```diff
@@ -49,6 +49,43 @@ public sealed partial class RestClient
                     _polygonHttpClient, FakeThrottler.Instance, builder, cancellationToken);
         }
 
+        /// <summary>
+        /// Gets list of historical trades for a single asset from Polygon's REST API endpoint.
+        /// </summary>
+        /// <param name="symbol">>Asset name for data retrieval.</param>
+        /// <param name="date">Single date for data retrieval.</param>
+        /// <param name="timestamp">Paging - Using the timestamp of the last result will give you the next page of results.</param>
+        /// <param name="timestampLimit">Maximum timestamp allowed in the results.</param>
+        /// <param name="limit">Limits the size of the response.</param>
+        /// <param name="reverse">Reverses the order of the results.</param>
+        /// <param name="cancellationToken">A cancellation token that can be used by other objects or threads to receive notice of cancellation.</param>
+        /// <returns>Read-only list of historical trade information.</returns>
+        public Task<IHistoricalItems<IHistoricalTradeV2>> ListHistoricalTradesV2Async(
```

Comment:
If we will extend the existing interface we can just make method with the same name and in case of same order or arguments it will be resolved by a C# compiler as a default candidate and users will be able to use new API w/o any code changes.